src/components/RecordsDetail/index.js

```diff
@@ -188,6 +193,18 @@ const RecordsDetail = props => {
       label={t({ comment: 'About minimap message', message: 'about minimap' })}
     /> : null
     }
+        <form className ='refine-search-form' >
+        <input type="hidden" name="category" value={curr_category} />
+        <input type="hidden" name="limit" value="40" />
+        <label for="query" className='refine-search-label'>Refining Search:</label>
+        <input className='refine-search-input'
+               type='text' 
+               id="query"
+               name="query"
+               placeholder={curr_query || ''}
+               maxLength={255}
+               size={60}/>
+    </form>
```

Comment:
I think the form should have a submit button for accessibility purposes.

Reply:
I agree with this!